babel/messages/pofile.py

```diff
@@ -249,7 +252,7 @@ def _finish_current_message(self) -> None:
         if self.messages:
             if not self.translations:
                 self._invalid_pofile("", self.offset, f"missing msgstr for msgid '{self.messages[0].denormalize()}'")
-                self.translations.append([0, _NormalizedString("")])
+                self.translations.append([0, _NormalizedString('""')])
```

Comment:
Wouldn't
```suggestion
                self.translations.append([0, _NormalizedString()])
```
do?

Reply:
Mmmm... Can't recall without looking at this in a computer and running tests, but I have vague memories of this needing a value so the `_NormalizedString` instance's `__bool__` method could return `True`

Reply:
It seems this was just me confusing myself with how `_NormalizedString` works. I've checked and I can safely (all tests run correctly) replace all instances of `_NormalizedString('""')` with `_NormalizedString()`, even a couple that weren't included in the initial changeset.